webroot/.htaccess

```diff
@@ -1,5 +1,6 @@
 <IfModule mod_rewrite.c>
     RewriteEngine On
-    RewriteCond %{REQUEST_FILENAME} !-f
-    RewriteRule ^ index.php [L]
+    RewriteBase   /
```

Comment:
This should be commented otherwise it will cause problem for sane hostings which don't need it.